css-media-queries/meta-viewport.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <meta name="viewport" content="width=300, initial-scale=1">
+  <!-- your code here 👇 -->
+
```

Comment:
Please tie the viewport meta tag to the screen's width:

---

![image](https://user-images.githubusercontent.com/16143771/170393046-bc4017b4-11f7-456c-acf9-041676c35a25.png)